src/Microsoft.ML.Transforms/CategoryImputer.cs

```diff
@@ -0,0 +1,1824 @@
+        {
+            private TransformerEstimatorSafeHandler _transformerHandler;
+
+            internal UInt32TypedColumn(string name, string source) :
+                base(name, source, typeof(uint).ToString())
+            {
+            }
+
+            [DllImport("Featurizers", EntryPoint = "CatImputerFeaturizer_uint32_t_CreateEstimator", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+            private static extern bool CreateEstimatorCpp(out IntPtr estimator, out IntPtr errorHandle);
+
+            [DllImport("Featurizers", EntryPoint = "CatImputerFeaturizer_uint32_t_DestroyEstimator", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+            private static extern bool DestroyEstimatorCpp(IntPtr estimator, out IntPtr errorHandle); // Should ONLY be called by safe handle
+
+            [DllImport("Featurizers", EntryPoint = "CatImputerFeaturizer_uint32_t_CreateTransformerFromEstimator", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+            private static extern bool CreateTransformerFromEstimatorCpp(TransformerEstimatorSafeHandler estimator, out IntPtr transformer, out IntPtr errorHandle);
+            internal override void CreateTransformerFromEstimator(IDataView input)
+            {
+                _transformerHandler = CreateTransformerFromEstimatorBase(input);
+            }
+
+            [DllImport("Featurizers", EntryPoint = "CatImputerFeaturizer_uint32_t_CreateTransformerFromSavedData", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+            private static extern bool CreateTransformerFromSavedDataCpp(IntPtr data, IntPtr bufferSize, out IntPtr transformer, out IntPtr errorHandle);
+            internal override void CreateTransformerFromSavedData(byte[] data)
+            {
+                bool result;
+                GCHandle handle = GCHandle.Alloc(data, GCHandleType.Pinned);
+                try
+                {
+                    IntPtr rawDataPtr = handle.AddrOfPinnedObject();
+                    IntPtr bufferSize = new IntPtr(data.Count());
+                    result = CreateTransformerFromSavedDataCpp(rawDataPtr, bufferSize, out IntPtr transformer, out IntPtr errorHandle);
+
+                    if (!result)
+                    {
+                        throw new Exception(GetErrorDetailsAndFreeNativeMemory(errorHandle));
+                    }
+                    _transformerHandler = new TransformerEstimatorSafeHandler(transformer, DestroyTransformerCpp);
+
+                }
+                finally
+                {
+                    handle.Free();
+                }
+            }
+
+            [DllImport("Featurizers", EntryPoint = "CatImputerFeaturizer_uint32_t_DestroyTransformer", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+            private static extern bool DestroyTransformerCpp(IntPtr transformer
```

Comment:
Don't these estimators need to be exported to onnx? Or is that functionality coming in later? 
We should chat about who does that work.

Reply:
Per offline discussion, resolving.